code/util/intradaybikeswd.q

```diff
@@ -0,0 +1,6 @@
+// Gets the rdb port past in from the command line.
+conn:.Q.def[(enlist `conn)!enlist 1;.Q.opt .z.x][`conn]
```

Comment:
is 1 a sensible default here? would it worthwhile making it a null, then error trapping below if the port isn't open/null/can't be connected to for whatever reason?